### DISCUSSION TITLE
Change organization from belongs to a business to my personal account

### DISCUSSION THREAD

**Discussion by Ltwoz** [community/community#75692] [New to GitHub]
Hi,
I was created organization and the form is a bit unclear for me.
So i fill out this field

![image](https://github.com/community/community/assets/87902742/2e5ccb81-439e-4572-920e-f0c88f674728)

and now i want to change it from belongs to a business -> to my personal account

I cannot find any setting for that in organization.

Thanks

**Comment by Dhruv00000** [comment]
You cannot change am organization's ownership from business to personal but you can create a new personal organization and add the repositories to it.
